groovy/postInit/mod/VanillaRecipes.groovy

```diff
@@ -322,6 +322,11 @@ crafting.replaceShaped('minecraft:torchAnthDust', item('minecraft:torch') *8, [
     [ore('stickWood')]
 ])
 
+crafting.addShaped('minecraft:torchCarbonDust', item('minecraft:torch') *8, [
```

Comment:
* 8

Reply:
elaborate more on this

Reply:
add a space bruh

Reply:
> add a space bruh

![image](https://github.com/SymmetricDevs/Supersymmetry/assets/51245052/cc8d35f6-ea00-4a3a-a3ec-6d27189cb2ca)

Reply:
Bro that is the problem

Reply:
The problem is there is no space